salt/metalk8s/addons/prometheus-operator/deployed/alertmanager-configuration-secret.sls

```diff
@@ -0,0 +1,16 @@
+
+{%- set alertmanager = salt.metalk8s_service_configuration.get_service_conf(
+        'metalk8s-monitoring', 'metalk8s-alertmanager-config'
+  )
+%}
+
+Patch Alertmanager Configuration Secret:
+  metalk8s_kubernetes.object_updated:
+    - name: alertmanager-prometheus-operator-alertmanager
+    - apiVersion: v1
+    - kind: Secret
+    - namespace: metalk8s-monitoring
+    - patch:
+        stringData:
+          alertmanager.yaml: |-
+            {{ alertmanager.spec.notification.config | tojson }}
```

Comment:
Does it work ??? because the `|-` will get the bellow block as a string where we expect a dict
For me you should do 
```suggestion
          alertmanager.yaml: {{ alertmanager.spec.notification.config | tojson }}
```
As here we control it, it's not from the template

Reply:
Ok no i'm wrong I was thinking it's a subkey of the yaml alertmanager but no the spec contain the full yaml so yes you right sorry for the noise